content/drone-plugins/drone-gcr/index.md

```diff
@@ -9,7 +9,7 @@ image: plugins/gcr
 ---
 
 {{% alert info %}}
-The GCR plugin is an extension around the Docker plugin. Please see the documentation for the [Docker plugin]({{< relref "drone-plugins/drone-docker/index.md" >}}) for other configuration options.
+The GCR plugin is an extension around the Docker plugin. Please see the documentation for the [Docker plugin]({{ relLangURL "/drone-plugins/drone-docker/index.md" }}) for other configuration options.
```

Comment:
Why relLangURL?

Reply:
Ah yeah, you are right... I forgot that the plugin index have been prepared to localization.